#PreHData.msg
int64 num_current_points
int64 num_matched_points
int64 score